tests/test_embedding.py

```diff
@@ -0,0 +1,67 @@
+import finalfusion
+import pytest
+
+NORM = [
+    6.557438373565674,
+    8.83176040649414,
+    6.164413928985596,
+    9.165151596069336,
+    7.4833149909973145,
+    7.211102485656738,
+    7.4833149909973145
+]
+
+
+def test_with_norms_embeddings():
+    embeds = finalfusion.Embeddings(
+        "tests/embeddings.fifu")
+    embeds_dict = dict()
+    with open("tests/embeddings.txt", "r", encoding="utf8")as lines:
+        for line in lines:
+            line_list = line.split(' ')
+            embeds_dict[line_list[0]] = line_list[1:]
+
+    with_norm_iter = embeds.iter_with_norm()
+    for idx_row, with_norm_embeds in enumerate(with_norm_iter):
+        for idx_col, normed_embed in enumerate(with_norm_embeds[1]):
+            unnormed_embed = "{0:.1f}".format(normed_embed * NORM[idx_row])
```

Comment:
This looks almost ready to go, but don't use strings for number comparisons. Use `pytest.approx` with the applicable tolerance.

Reply:
But even better: multiply the embedding by its norm, then use `numpy.allclose`.